src/models/index.js

```diff
@@ -28,48 +28,50 @@ Tag.addScope('orderedByRestaurant', {
 });
 
 Restaurant.findAllWithTagIds = ({ team_id }) =>
-  Restaurant.findAll({
-    attributes: {
+  Team.findById(team_id).then(team =>
```

Comment:
It might be worth trying to add an additional `sequelize.literal` that does something like "select vote_duration from teams where team_id", instead of wrapping this DB query in another resolved DB query. I honestly don't know enough about SQL to determine whether that's possible, but it seems like you'd be able to use subquery results to affect subsequent subqueries.

Reply:
Yeah, I was thinking about that.  I'll look into it and also read up a little to see which might be more performant.

Reply:
Hey Jeffrey, it doesn't appear that you can wrap literals inside of each other like that.  I can use the `sequelize.query` method to run a raw SQL command, but we still have to wait for it to resolve before its results can be used in the subqueries.  I think it's cleaner to let sequelize build and use its model for a Team, but I'm curious to hear what you think.

Reply:
Yeah, let's go with that then. I think there might be an option to only return a specific field while performing the query which might speed it up a bit, but otherwise, let's keep it as a separate call.

Reply:
Okay, I'll leave this code as-is, then.